lib/api/objectHead.js

```diff
@@ -87,10 +127,8 @@ function objectHead(authInfo, request, log, callback) {
                     return callback(errors.InvalidRange, corsHeaders);
                 }
                 // eslint-disable-next-line no-param-reassign
-                objMD['content-length'] = partSize;
+                responseHeaders['content-length'] = partSize;
```

Comment:
minor, but you I don't think you need the eslint comment above this line anymore

Reply:
good catch